ur_robot_driver/include/ur_robot_driver/ros/hardware_interface.h

```diff
@@ -156,6 +161,19 @@ class HardwareInterface : public hardware_interface::RobotHW
   bool shouldResetControllers();
 
 protected:
+  /*!
+   * \brief Applies joint limits & soft joint limits on position, velocity & effort defined in URDF / parameter server
+   *
+   */
+  void registerJointLimits(ros::NodeHandle& robot_hw_nh, const hardware_interface::JointHandle& joint_handle_position,
```

Comment:
Does this function apply the limits or register the limits? I guess, the docstring needs to be adapted.